src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -64,13 +64,13 @@ def load_arguments(self, _):
         c.argument('backup_name', help='The name of the backup file to create.')
         c.argument('storage_account_name', arg_group='Storage', help='The name of the storage account used to place the backup.')
         c.argument('storage_account_key', arg_group='Storage', help='The access key of the storage account used to place the backup.')
-        c.argument('storage_account_container', options_list=['--storage-account-container', '--container-name'], nargs='+', help='The name of the storage account container used to place the backup.')
```

Comment:
It's a breaking change on CLI command interface. Please provide more information to support this change.

Reply:
issue: https://github.com/Azure/azure-cli/issues/16262, this cli command already broken, need to fix it